README.md

```diff
@@ -6,18 +6,11 @@ ImageDePHI is an application to redact personal data (PHI) from whole slide imag
 ## Installation
 * Download the [latest ImageDePHI release](https://github.com/DigitalSlideArchive/ImageDePHI/releases/latest).
 
-* On **Linux and macOS only**, [make the file executable](https://askubuntu.com/a/229592):
-    ```bash
-    chmod +x ./Downloads/imagedephi-Linux
-    ```
-    or
-    ```bash
-    chmod +x ./Downloads/imagedephi-macOS
-    ```
+* Unzip the downloaded file, which will extract the executable named `imagedephi` (or `imagedephi.exe` on Windows).
```

Comment:
This is great, it fixes an unexpected behavior on windows: the terminal flashing open and immediately closing with the command `start imagedephi.exe`
I do however think that the user still needs to make it executable for linux and mac though.

Reply:
> I do however think that the user still needs to make it executable for linux and mac though.

Have you verified this? It should no longer be necessary because the zip file should preserve the file permissions.